docs/properties/recommended_optional/property_distribution.rst

```diff
@@ -31,30 +29,47 @@ Collections of files should be either using an archive format or a BagIt folder
      </distribution>
    </distributions>
 
-.. _21.a:
 
-21.a mediaType
+.. _21.1:
+
+21.1 file
```

Comment:
The use of Distribution for a “bag-of-files” (which seems to be the issue of discussion in this PR) is huge in the DCAT community. See: 
	●	https://github.com/w3c/dxwg/issues/317
	●	https://w3c.github.io/dxwg/dcat/#examples-bag-of-files
	●	https://github.com/w3c/dxwg/issues/63
	●	https://github.com/w3c/dxwg/issues/256
	●	https://www.w3.org/2018/07/19-dxwgdcat-minutes#x09

It seems to be that they solution was discourage the use of distribution for this cases and instead us relation
> use `dct:relation` for unstructured collections

Additionally they give some liberty to the metadata provider [here](https://www.w3.org/TR/vocab-dcat-2/#Class:Distribution) with:
> the question of whether different representations can be understood to be distributions of the same dataset, or distributions of different datasets, is application specific. Judgement about how to describe them is the responsibility of the provider, taking into account their understanding of the expectations of users, and practices in the relevant community.

Reply:
We heard in the RFC that the use case where a distribution has multiple files is significant. Rather than requiring that these users either 1) use bagit/a container format or 2) inaccurately represent these components of a distribution as separate distributions, I think we can accommodate them with this change. In the case where a metadata provider wants to use a bag, they can represent that as a single file.

Reply:
Add revised version of: "Judgement about how to describe them is the responsibility of the provider, taking into account their understanding of the expectations of users, and practices in the relevant community."